imap_processing/codice/utils.py

```diff
@@ -140,7 +144,10 @@ def create_hskp_dataset(packets) -> xr.Dataset:
 
     dataset = xr.Dataset(
         coords={"epoch": epoch},
-        attrs=cdf_attrs.l1a_hskp_attrs.output(),
+        attrs={
+            **cdf_attrs.get_global_attributes(),
+            **cdf_attrs.global_attributes["imap_codice_l1a_hskp"],
```

Comment:
Is there a better way to do this? I want to be able to get one dictionary that contains the default global attributes (i.e. `Project`, `Source_name`, etc.) with the instrument- and data product-specific attributes (i.e. `Data_type`, `Logical_source`, etc.)

Reply:
Yes there is, made a comment